modules/cert-manager-acme-challenges-types.adoc

```diff
@@ -12,6 +12,13 @@ HTTP-01:: With the HTTP-01 challenge type, you provide a computed key at an HTTP
 +
 For more information, see link:https://cert-manager.io/docs/configuration/acme/http01/[HTTP01] in the upstream cert-manager documentation.
 
+[NOTE]
+====
+HTTP-01 requires _Let's Encrypt_ servers to access the cluster's route. Note that using internal or private clusters behind the proxy to access, would fail the HTTP-01 validation checks required to issue a certificate.
+
+The HTTP-01 challenge is restricted to port 80 only. Allowing clients to specify arbitrary ports would make the challenge less secure, and therefore it is not allowed by the ACME standard.
```

Comment:
```suggestion
The HTTP-01 challenge is restricted to port 80. Allowing clients to specify arbitrary ports would make the challenge less secure, and therefore it is not allowed by the ACME standard.
```
Is the second sentence required?

Reply:
Specifying why it wouldn't work would be helpful to the users.

Reply:
That does make sense. Let me check with QE once if we can remove the sentence. Thanks for the review.

Reply:
@maxwelldb I can remove the sentence and add this link for reference- https://letsencrypt.org/docs/challenge-types/#http-01-challenge. It contains more information about Lets Encrypt. wdyt?

Reply:
Totally up to you. I only parachuted in for peer review last week--I'm not aware of the context or customer needs of people who would be using this content.

Reply:
Thank you.